### PR TITLE
Enable periodic task execution for OAI End-to-End testing

### PR DESCRIPTION
**Pull request by electrocucaracha**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespaces from that line:
>
> /kind bug
> /kind cleanup
> /kind design
> /kind documentation
> /kind failing-test

/kind feature

> /kind flake

**What this PR does / why we need it**:
Until now, OAI e2e execution is done on demand basis. That restriction limits the capability for catching errors on new changes. This change enables the End-to-End test execution for OAI use case.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

**Comment by arora-sagar** [comment]
@electrocucaracha What is the benefit of doing this? Sorry I am not aware why are we doing this?

**Comment by electrocucaracha** [comment]
> @electrocucaracha What is the benefit of doing this? Sorry I am not aware why are we doing this?

This will execute the OAI end-to-end tests on a daily basis

File: prow/config/config.yaml [hunk 1]
```diff
@@ -1341,3 +1341,58 @@ periodics:
             items:
               - key: nephio.yaml
                 path: nephio.yaml
+  - name: e2e-daily-ubuntu-jammy-oai
+    annotations:
+    labels:
+    cron: "0 15 * * 1-5"
```
Comment (radoslawc) [inline prow/config/config.yaml:1347]:
```suggestion
    cron: "0 15 * * 1-6"
```

**Pull request merged**